packages/playground/windows/playground-win32/Playground-win32.vcxproj

```diff
@@ -169,7 +169,7 @@
     <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.targets" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.targets')" />
   </ImportGroup>
   <ItemGroup>
-    <PackageReference Include="boost" Version="1.76.0.0" />
```

Comment:
><PackageReference Include="boost" Version="1.76.0.0" /> [](http://example.com/codeflow?start=4&length=55)

Does the UWP Playground app need a similar update, or does it use a different network stack?

Reply:
Both UWP and Desktop use the same stack.